tutor/test_t1_28_WorkAReading.py

```diff
@@ -1511,5 +1543,81 @@ def test_student_verify_started_assignment_shows_in_progress_on_dashboard(self):
+
+                assert('Long Automation Reading' in name.text), \
+                    'Not viewing the reading'
+
+                while(1):
+                    while ('arrow right' in self.student.driver.page_source):
+                        self.student.find(By.XPATH, "//a[contains(@class,'arrow') and contains(@class,'right')]").click()
+
+                    # multiple choice case
+                    if('exercise-multiple-choice' in self.student.driver.page_source):
+                        
+
+                        answers = self.student.driver.find_elements(By.CLASS_NAME, 'answer-letter')
+                        self.student.sleep(0.8)
+                        rand = randint(0, len(answers) - 1)
+                        answer = chr(ord('a') + rand)
+                        #print('Selecting %s' % answer)
+                        Assignment.scroll_to(self.student.driver, answers[0])
+                        if answer == 'a':
+                            self.student.driver.execute_script('window.scrollBy(0, -160);')
+                        elif answer == 'd':
+                            self.student.driver.execute_script('window.scrollBy(0, 160);')
+                        answers[rand].click()
+
+
+                        self.student.wait.until(
+                            expect.element_to_be_clickable(
+                                (By.XPATH, '//button[contains(@class,"async-button") and contains(@class,"continue")]')
+                            )
+                        ).click()
+                        self.student.sleep(5)
+
+                        assert('question-feedback bottom' in self.student.driver.page_source), \
+                            'Did not submit MC'
+
+                        break
+
+                    # free response case
+                    elif('textarea' in self.student.driver.page_source):
+                        self.student.find(By.TAG_NAME, 'textarea').send_keys('An answer for this textarea')
+                        self.student.sleep(2)
+                        self.student.wait.until(
+                            expect.element_to_be_clickable(
+                                (By.XPATH, '//button[contains(@class,"async-button") and contains(@class,"continue")]')
+                            )
+                        ).click()
+
+                        self.student.wait.until(
+                            expect.visibility_of_element_located(
+                                (By.CLASS_NAME, 'exercise-multiple-choice')
+                            )
+                        )
+
+                        self.student.sleep(2)
+
+                self.student.sleep(20)
+                self.student.driver.get(home)
+
+                assignments = self.student.driver.find_elements_by_xpath("//div[@class='-upcoming panel panel-default']/div[@class='panel-body']/div[@class='task row reading workable']")
```

Comment:
Are you checking your code against PEP8 standards (in this case a line should not exceed 79 characters in length)?